gemfiles/Gemfile.rails-3.2.x

```diff
@@ -4,3 +4,4 @@ source "http://rubygems.org"
 
 gemspec path: '..'
 gem 'rails', "~> 3.2.13"
+gem 'test-unit', '~> 3.0'
```

Comment:
on latest versions of ruby, test unit is removed. so i explicity added that gem for rails 3.x